metricbeat/module/docker/diskio/diskio_test.go

```diff
@@ -93,6 +93,75 @@ func TestDeltaMultipleContainers(t *testing.T) {
 
 }
 
+func TestBlkIOSkip(t *testing.T) {
+	/*
+	   For context, here's what a "raw" event looks like coming in from the docker stats API.
+	   Note the repeated values across the two different major devices. Here 8:0 is a /dev/sda disk, 253:0 is the device mapper that exposes that disk's storage space.
+	           "io_service_bytes_recursive": [
+	               {
+	                   "major": 8,
+	                   "minor": 0,
+	                   "op": "read",
+	                   "value": 2359296
+	               },
+	               {
+	                   "major": 8,
+	                   "minor": 0,
+	                   "op": "write",
+	                   "value": 94544896
+	               },
+	               {
+	                   "major": 253,
+	                   "minor": 0,
+	                   "op": "read",
+	                   "value": 2359296
+	               },
+	               {
+	                   "major": 253,
+	                   "minor": 0,
+	                   "op": "write",
+	                   "value": 94544896
+	               }
+	           ],
+	*/
+
+	var readVal uint64 = 2359296
+	var writeVal uint64 = 94544896
+	testInt := []types.BlkioStatEntry{
+		{
+			Major: 8,
+			Minor: 0,
+			Op:    "Read",
+			Value: readVal,
+		},
+		{
+			Major: 8,
+			Minor: 0,
+			Op:    "Write",
+			Value: writeVal,
+		},
+		{
+			Major: 253,
+			Minor: 0,
+			Op:    "Read",
+			Value: readVal,
+		},
+		{
+			Major: 253,
+			Minor: 0,
+			Op:    "Write",
+			Value: writeVal,
+		},
+	}
+
+	skip := []uint64{253}
```

Comment:
Let's also test the case where `skip = []uint64{}` here to ensure the behaviour is different between the two cases. Perhaps with a comment that the empty skip case is an incorrect result.